farcaster/hubs/values.yaml

```diff
@@ -62,27 +60,28 @@ service:
         port: 2283
 
 # for use with Kubernetes Gateway API
-route:
-  hubble: 
-    enabled: true
-    kind: HTTPRoute
-    parentRefs:
-      - name: gateway
-        namespace: gateway-namespace
-    hostnames:
-      - hub.dtech.vision
-    rules:
-      - matches: 
-        - path:
-            type: Prefix
-            value: /
-        backendRefs:
-          - kind: Service
-            name: hubble    
+# route:
+#   hubble:
+#     enabled: true
+#     kind: HTTPRoute
+#     parentRefs:
+#       - name: gateway
+#         namespace: gateway-namespace
+#     hostnames:
+#       - hub.dtech.vision
+#     rules:
+#       - matches:
+#         - path:
+#             type: Prefix
+#             value: /
+#         backendRefs:
+#           - kind: Service
+#             name: hubble
```

Comment:
tbqh i am not sure what this is doing? i havent used kube gateway api, do we need this? seems like it might be better served by an Ingress but idk

Reply:
yeah could also just have users deploy their own gateway route ressource and not have it be the default.